config.json

```diff
@@ -469,6 +469,22 @@
                 "integers",
                 "math"
             ]
+        },
+        {
+            "uuid": "fcb9ef0e-ed97-4420-a38f-9bf690c23a3b",
+            "slug": "dnd-character",
+            "core": false,
+            "unlocked_by": "robot-name",
+            "difficulty": 3,
```

Comment:
`robot-name` unlocks this exercise and has a difficulty of 5, so I think should be at least as high as `robot-name`